src/Raven.Server/Documents/Indexes/Static/TimeSeries/MapTimeSeriesIndex.cs

```diff
@@ -279,8 +297,14 @@ private void HandleTimeSeriesChange(TimeSeriesChange change)
         {
             progressStats.NumberOfItemsToProcess +=
                 DocumentDatabase.DocumentsStorage.TimeSeriesStorage.GetNumberOfTimeSeriesSegmentsToProcess(
-                    queryContext.Documents, collectionName, progressStats.LastProcessedItemEtag, out var totalCount,overallDuration);
+                    queryContext.Documents, collectionName, progressStats.LastProcessedItemEtag, out var totalCount, overallDuration);
             progressStats.TotalNumberOfItems += totalCount;
+
+
+            progressStats.NumberOfTombstonesToProcess +=
```

Comment:
we have regular tombstones and timeseries delete ranges? we need to take both into account?

Reply:
done